README.md

```diff
@@ -98,3 +98,12 @@ Before starting any work, please either comment on an existing issue, or file a
 ## Supportability
 
 Currently, Kubebuilder officially supports OSX and Linux platforms. So, if you are using a Windows OS you may find issues. Contributions towards supporting Windows are welcome.
+
+## Community Meetings
+
+The following meetings happens biweekly. 
+- [Kubebuilder, Controller Runtime, and Controller Tools](https://docs.google.com/document/d/1Ih-2cgg1bUrLwLVTB9tADlPcVdgnuMNBGbUl4D-0TIk/edit#heading=h.dp4gt7gj3cn) 
+- Kubebuilder Triage
```

Comment:
What meeting does this refer to? Don't we only have the shared meeting above?

Reply:
Because of this, the PR is /hold. It will happen soon.

Reply:
Add in a follow-up when this meeting is set up?

Reply:
The doodle was sent today. Up to next Friday, it will be setup. 
So, my idea is to keep it in hold until there. Are you OK with? WDYT?